description: 'misc/libmasm/module: Fix two use after free in ibmasm_init_one'
references:
- https://www.cve.org/CVERecord?id=CVE-2021-47334
- https://git.kernel.org/stable/c/1512e7dc5eb08b7d92a12e2bfcd9cb8c4a1ec069
- https://git.kernel.org/stable/c/29ba8e2ba89ee2862a26d91204dd5fe77ceee25a
- https://git.kernel.org/stable/c/5b06ca113bf197aab2ab61288f42506e0049fbab
- https://git.kernel.org/stable/c/481a76d4749ee3a27f902ba213fdcbb4bb39720e
- https://git.kernel.org/stable/c/38660031e80eaa6cc9370b031c180612f414b00d
- https://git.kernel.org/stable/c/b9c87ce3bc6331f82811a8cf8e930423c22523a3
- https://git.kernel.org/stable/c/ef1067d2baa847d53c9988510d99fb494de4d12c
- https://git.kernel.org/stable/c/a7268e8a227d5a4f0bd1584f556246b0224ab274
- https://git.kernel.org/stable/c/7272b591c4cb9327c43443f67b8fbae7657dd9ae
- https://lore.kernel.org/linux-cve-announce/2024052135-CVE-2021-47334-433e@gregkh/
- https://git.kernel.org/linus/7272b591c4cb9327c43443f67b8fbae7657dd9ae
comments:
  cip/cip-kernel-sec: There is no mention of the commit that introduced the bug in
    the CVE announcement.Fixed in v5.14-rc1.
  debian/carnil: First introducing commit could not be determined.
fixed-by:
  cip/4.19: [481a76d4749ee3a27f902ba213fdcbb4bb39720e]
  cip/4.19-rt: [481a76d4749ee3a27f902ba213fdcbb4bb39720e]
  cip/4.4: [1512e7dc5eb08b7d92a12e2bfcd9cb8c4a1ec069]
  cip/4.4-rt: [1512e7dc5eb08b7d92a12e2bfcd9cb8c4a1ec069]
  cip/4.4-st: [1512e7dc5eb08b7d92a12e2bfcd9cb8c4a1ec069]
  cip/5.10: [b9c87ce3bc6331f82811a8cf8e930423c22523a3]
  cip/5.10-rt: [b9c87ce3bc6331f82811a8cf8e930423c22523a3]
  debian/buster-security: ['version:debian/4.19.208-1']
  mainline: [7272b591c4cb9327c43443f67b8fbae7657dd9ae]
  stable/4.19: [481a76d4749ee3a27f902ba213fdcbb4bb39720e]
  stable/4.4: [1512e7dc5eb08b7d92a12e2bfcd9cb8c4a1ec069]
  stable/5.10: [b9c87ce3bc6331f82811a8cf8e930423c22523a3]
  stable/5.4: [38660031e80eaa6cc9370b031c180612f414b00d]
  ubuntu/bionic: [bf70c57137f36bc7e8ca98c84136429947e9c4f8]
  ubuntu/focal: [85f30a9f57c90b4b3114238752805926386f29f1]
